go/bss-core/metrics/interface.go

```diff
@@ -0,0 +1,38 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+type Metrics interface {
+	// SubsystemName returns the subsystem name for the metrics group.
+	SubsystemName() string
+
+	// BalanceETH tracks the amount of ETH in the submitter's account.
+	BalanceETH() prometheus.Gauge
+
+	// BatchSizeBytes tracks the size of batch submission transactions.
+	BatchSizeBytes() prometheus.Summary
+
+	// NumElementsPerBatch tracks the number of L2 transactions in each batch
+	// submission.
+	NumElementsPerBatch() prometheus.Summary
+
+	// SubmissionTimestamp tracks the time at which each batch was confirmed.
```

Comment:
`batch` -> `transaction`

Reply:
I think batch is still accurate? We do have support for batched withdrawals. IMO "batch" is still correct, even in the degenerate case of 1 element